src/Auth0.OidcClient.Core/Auth0ClientBase.cs

```diff
@@ -119,7 +119,7 @@ private OidcClientOptions CreateOidcClientOptions(Auth0ClientOptions options)
                 Flow = AuthenticationFlow.AuthorizationCode,
                 ResponseMode = _formPost ? AuthorizeResponseMode.FormPost : AuthorizeResponseMode.Redirect,
                 RedirectUri = options.RedirectUri ?? $"https://{_options.Domain}/mobile",
-                PostLogoutRedirectUri = options.PostLogoutRedirectUri,
+                PostLogoutRedirectUri = options.PostLogoutRedirectUri ?? $"https://{_options.Domain}/mobile",
```

Comment:
should we add a test for this case?

Reply:
There are no unit tests today for this library.